tests/test_read.py

```diff
@@ -239,7 +240,7 @@ def test_barebones():
 
 def test_barebones_missing_all_sections():
     las = lasio.read(egfn("barebones2.las"))
-    assert las.curves[-1].mnemonic == "UNKNOWN:8"
+    assert las.sections == {}
```

Comment:
This needs to go back to the old test, I want to find a way to retain this behaviour -- I'll work it out.